moabb/utils.py

```diff
@@ -9,6 +10,7 @@
 from typing import TYPE_CHECKING
 
 import numpy as np
+from docstring_inheritance import NumpyDocstringInheritanceMeta
```

Comment:
```suggestion
from docstring_inheritance import NumpyDocstringInheritanceInitMeta
```
I think we need to use this class instead